src/seedu/addressbook/commands/ExitCommand.java

```diff
@@ -16,4 +16,8 @@ public CommandResult execute() {
         return new CommandResult(MESSAGE_EXIT_ACKNOWEDGEMENT);
     }
 
+    @Override
+    public boolean isMutating() {
```

Comment:
Remember to update your method with header comments as your overriden method has a different behaviour from the parent class implementation. e.g. Returns if true if class mutates data.